.eslintrc.js

```diff
@@ -46,43 +46,44 @@ module.exports = {
     "jest/no-disabled-tests": 1,
     "jest/no-focused-tests": 1,
     "jest/no-identical-title": 2,
-    "no-empty": 0,
```

Comment:
sorry for the noise, I sorted this list to make sure `space-in-parens` wasn't already listed